CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix issue where VF stops working after changing device while muted
```

Comment:
"VF" -> "Amazon Voice Focus", and add comma end of line.

Reply:
Changed